source/_components/emulated_roku.markdown

```diff
@@ -0,0 +1,114 @@
+---
+layout: page
+title: "Emulated Roku"
+description: "Instructions on how to set up Emulated Roku within Home Assistant."
+date: 2018-10-03 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Hub
+ha_release: 0.80.0
+ha_iot_class: "Local Push"
+ha_qa_scale: internal
+---
+
+This component integrates an emulated Roku API into Home Assistant,  
+so remotes such as Harmony and Android apps can connect to it through WiFi as it were a Roku player.  
+Home Assistant will see key presses and app launches as Events, which you can use as triggers for automations.  
+Multiple Roku servers may be started if you run out of buttons by specifying multiple server entries.  
+When using multiple servers, you can filter the roku USN which is specified via the `name` attribute.
+
+<p class='note'>  
+Windows is not supported because Home Assistant uses `ProactorEventLoop` which does not support UDP sockets.
+</p>  
+
+## {% linkable_title Configuration %}
+
+configuration.yaml
+```yaml
+# Example configuration.yaml entry
+emulated_roku:
+  servers:
+    - name: hass roku
+      listen_port: 8060
+      host_ip: 192.168.1.150
+      advertise_ip: 10.0.0.10
+      advertise_port: 8070
+      upnp_bind_multicast: True
+```
+
+{% configuration %}
+name:
+  description: Name of the roku that will be displayed as USN in Harmony.
+  required: true
+  type: string
+listen_port:
+  description: The port the roku API will run on. This can be any free port on your system.
+  required: true
+  type: integer
+host_ip:
+  description: The IP address that your Home Assistant installation is running on. If you do not specify this option, the component will attempt to determine the IP address on its own.
+  required: false
+  type: string
+advertise_ip:
+  description: If you need to override the IP address used for UPnP discovery. (For example, using network isolation in Docker)
+  required: false
+  type: string
+advertise_port:
+  description: If you need to specifically override the advertised UPnP port.
```

Comment:
```suggestion
  description: If you need to override the advertised UPnP port specifically.
```

Reply:
emulated_hue has the same description. I'd rather just remove the word, it's superfluous and the other property descriptions do not use it.